packages/playground-server/src/api.ts

```diff
@@ -23,6 +25,18 @@ export default function mountApi() {
   return api;
 }
 
+export function createErrorResponseForDatabase(
```

Comment:
can we move this elsewhere?

Reply:
Done via d404f19. We now have a proper error middleware, so no more try...catch needed in the controllers. This function is no longer necessary.